app/src/main/java/org/oppia/android/app/settings/profile/ProfileListFragmentPresenter.kt

```diff
@@ -19,7 +19,8 @@ import javax.inject.Inject
 class ProfileListFragmentPresenter @Inject constructor(
   private val activity: AppCompatActivity,
   private val fragment: Fragment,
-  private val viewModelProvider: ViewModelProvider<ProfileListViewModel>
+  private val viewModelProvider: ViewModelProvider<ProfileListViewModel>,
+  private val singleTypeAdapterBuilder: BindableAdapter.SingleTypeBuilder.Factory
```

Comment:
For consistency: `private val singleTypeBuilderFactory: `

Reply:
Updated.